easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.4rc1-GCC-11.3.0.eb

```diff
@@ -0,0 +1,39 @@
+name = 'OpenMPI'
+version = '4.1.4rc1'
+
+homepage = 'https://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+sources = [SOURCELOWER_TAR_BZ2]
+patches = ['OpenMPI-4.1.3_build_with_internal_cuda_header.patch']
+checksums = [
+    '46169edd9cf8366608684e5c6a342650b2393d46c9f9a380add8e09f1f8001de',  # openmpi-4.1.4rc1.tar.bz2
+    # OpenMPI-4.1.3_build_with_internal_cuda_header.patch
+    '53bff4fcc92350128b17c29d7288391f87f082f45645c631916b90d2fa5b5116',
+]
+
+builddependencies = [
```

Comment:
I think we need autotools here.

I was testing this on OpenMPI-4.1.1 and hit
```
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /local/EB/build/OpenMPI/4.1.1/GCC-11.2.0/openmpi-4.1.1/config/missing aclocal-1.15 -I config
/local/EB/build/OpenMPI/4.1.1/GCC-11.2.0/openmpi-4.1.1/config/missing: line 81: aclocal-1.15: command not found
```

Reply:
yup, added